lib-compose/src/main/java/com/what3words/components/compose/maps/W3WMapDefaults.kt

```diff
@@ -162,14 +162,14 @@ object W3WMapDefaults {
     }
 
     fun defaultMarkerConfig(
-        selectedZoomOutColor: W3WMarkerColor = MARKER_COLOR_DEFAULT,
-        multiMarkersColor: W3WMarkerColor = MUlTI_LIST_MARKERS_COLOR_DEFAULT,
+        selectedZoomOutColor: MarkerColor = SELECTED_ZOOM_OUT_MARKER_COLOR_DEFAULT,
+        defaultMarkerColor: MarkerColor = MARKER_COLOR_DEFAULT,
         selectedColor: Color = Color.Black,
         selectedColorDarkMode: Color = Color.White
     ): MarkerConfig {
         return MarkerConfig(
             selectedZoomOutColor = selectedZoomOutColor,
-            multiListMarkersColor = multiMarkersColor,
+            defaultMarkerColor = defaultMarkerColor,
```

Comment:
I think this can easily get confusing. We need 2 pair of colours, multilist (day, night) and selectedaddress (day,night).